src/index.mjs

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable no-process-exit */
+/* eslint no-console: 'off' */
+
+import chalk from 'chalk';
+import detectPackageManager from 'which-pm-runs';
+import { execa } from 'execa';
+import fs from 'fs';
+import ora from 'ora';
+
+const DEFAULT_CONFIG_FILE = '.stylelintrc.json';
+
+const STYLELINT_CONFIG_PATHS = new Set([
+	'.stylelintrc',
```

Comment:
[suggest] Adding also `.cjs` seems useful:

```suggestion
	'.stylelintrc',
	'.stylelintrc.cjs',
```

Reply:
Potentially additional files and paths based on cosmiconfig: 

https://github.com/davidtheclark/cosmiconfig/#cosmiconfig

- "any of the above two inside a `.config` subdirectory"
- "a `.config.js` or `.config.cjs` CommonJS module"